src/test/java/wooteco/subway/acceptance/favorite/FavoriteAcceptanceTest.java

```diff
@@ -0,0 +1,114 @@
+package wooteco.subway.acceptance.favorite;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import wooteco.subway.AcceptanceTest;
+import wooteco.subway.DummyTestUserInfo;
+import wooteco.subway.domain.favorite.FavoriteRepository;
+import wooteco.subway.infra.JwtTokenProvider;
+import wooteco.subway.service.favorite.dto.FavoriteResponse;
+import wooteco.subway.service.favorite.dto.FavoritesResponse;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Sql("/truncate.sql")
+public class FavoriteAcceptanceTest extends AcceptanceTest {
+    @Autowired
+    JwtTokenProvider jwtTokenProvider;
+
+    @Autowired
+    FavoriteRepository favoriteRepository;
+
+    @BeforeEach
+    public void setUp() {
+        favoriteRepository.deleteAll();
+    }
+
+    @DisplayName("즐겨 찾기 추가/삭제/조회")
+    @Test
+    void favoriteAcceptanceTest() {
+        /**
+         * 강남 - 역삼 - 선릉
+         * |           |
+         * |          한티
+         * |           |
+         * 양재 - 매봉 - 도곡
+         */
+        String token = jwtTokenProvider.createToken(DummyTestUserInfo.EMAIL);
+        //Given
+        // 지하철과 노선, 간선이 주어진다.
+        initStation();
+
+        //When
+        // 검색한 경로를 즐겨찾기에 추가하는 요청을 보낸다.
+        createMember(DummyTestUserInfo.EMAIL, DummyTestUserInfo.NAME, DummyTestUserInfo.PASSWORD);
+        createFavorite("강남역", "한티역", token);
+        createFavorite("매봉역", "도곡역", token);
+
+        //Then
+        // 즐겨찾기에 추가된다.
+        FavoritesResponse favoritesResponse = getAllFavorite(token);
+        List<FavoriteResponse> favoriteResponses = favoritesResponse.getFavoriteResponses();
+        assertThat(favoriteResponses).hasSize(2);
+        //assertThat(favoritesResponse.getSize()).isEqualTo(2);
+        //When
+        // 즐겨 찾기 목록중 하나를 삭제 한다.
+        FavoriteResponse favoriteResponse = favoriteResponses.get(0);
+        deleteFavorite(favoriteResponse, token);
+        //Then
+        // 즐겨 찾기 목록이 하나 줄어든다.
+        favoritesResponse = getAllFavorite(token);
+        assertThat(favoritesResponse.getFavoriteResponses()).hasSize(1);
+    }
+
+    private void deleteFavorite(FavoriteResponse favoriteResponse, String token) {
+        given().
+                header("Authorization", "Bearer " + token).
```

Comment:
```suggestion
                header(AuthorizationExtractor.AUTHORIZATION, "Bearer " + token).
```

가능하면 의미있는 상수를 사용하는 습관을 들여주세요~

Reply:
오! 좋은 팁 감사합니다 매번 상수화를 하는 것이 좋을까 고민도 했었거든요!
모두 수정하였습니다!